### ISSUE TITLE
[PATCH] option to twistd to save profile output as CSV file

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11226]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#263|
|Type|enhancement|
|Created|2003-09-17 21:46:11Z|

Attachments:

* [csvstats.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/065/065f8e41a20c940689359644aae39608d126c498/b94e3cdf74ead2bda8dd2b8981c92bc81f0230e7.patch) (2635 bytes) - added by headly on 2004-10-20 09:53:15Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__263 263
type__enhancement enhancement
reporter__itamarst itamarst
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__core core
keywords__ 
time__1063835171000000 1063835171000000
changetime__1264693124000000 1264693124000000
version__ 
owner__ 
cc__glyph cc__itamarst cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
This would make loading info into spreadsheets even easier.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="headly's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|headly commented|
|-|-|

```
#!html
<pre>
patch attached. Also fixes a bug when using the old style profiler.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

This patch needs unit tests before we can include it into the trunk.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

There's an option to save the "pstats" object now.  This is even better than a csv file.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
